### PR TITLE
Added giving schedules to dashboard

### PR DESCRIPTION
**Pull request by JakeDawkins**
Added the schedules list to the dashboard

**Still Needed**
- I need to add the link to the button
- I still need the link to the edit schedule

**Changes**
- I added the section header to the `SchedulesList` component so I wouldn't render it if there were no schedules. I need to do this to the activity portion of the dashboard as well

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://917590c4-783b-4618-b1b1-5126ff64074e.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/add-schedules/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: imports/pages/give/home/Schedules.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  render() {
+    if (!this.props.schedules || !this.props.schedules.scheduledTransactions) return <div />;
+    return (
+      <div>
+        <SectionHeader
+          title="Schedule"
+          link={
+            <SmallButton
+              text="New Schedule"
```
Comment (jbaxleyiii) [inline imports/pages/give/home/Schedules.js]: @JakeDawkins small optimization ability here. If you make this small button it's own component in the file, (i.e. a const reference) it won't get re instantiated each render

Comment (JakeDawkins) [inline imports/pages/give/home/Schedules.js]: 👍 Additional question: Should the card or the component rendering the card be in charge of formatting the date? On my activity card, I do it in the card component. The Schedule overview component assumes it's already formatted. Just trying to stay consistent

**Comment by jbaxleyiii** [comment]
The card should decide when possible. Since the formatting is purely
presentational which is the focus of the card

File: imports/pages/give/home/Schedules.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        frequency={schedule.schedule.description}
+        started={schedule.start}
+        latest={schedule.transactions[0] ? schedule.transactions[0].date : ""}
+        onEditClick={() => {}}
+      />
+    );
+  }
+
+  render() {
+    if (!this.props.schedules || !this.props.schedules.scheduledTransactions) return <div />;
```
Comment (jbaxleyiii) [inline imports/pages/give/home/Schedules.js]: @JakeDawkins we can just return `null` here instead of an empty div. Unless you had a reason for the div?

Comment (jbaxleyiii) [inline imports/pages/give/home/Schedules.js]: This also brings up a potential issue. @samclaridge how do we want to do loading states on the dashboard?

Comment (jbaxleyiii) [inline imports/pages/give/home/Schedules.js]: @samclaridge that is a great point! And yeah agreed on all accounts

**Pull request merged**